RELEASE_NOTES.md

```diff
@@ -1,6 +1,8 @@
 ### Version 2.6.7 (TBD)
 
 * Upgrade to Gradle 2.11.
+* Bug fix when copying single file from container and hostPath is set to directory for `DockerCopyFileFromContainer` - [Pull request 163](https://github.com/bmuschko/gradle-docker-plugin/pull/163)
+* Step reports are now printed to stdout by default for `DockerBuildImage` - [Pull request 145](https://github.com/bmuschko/gradle-docker-plugin/pull/145)
```

Comment:
If we go for `logger.quiet` then it should be done for all tasks for consistency.